app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,558 @@
+    "https://m.media-amazon.com/images/M/MV5BNzMyMTM1MjQxNF5BMl5BanBnXkFtZTgwMjY4NTE5NjE@._V1_.jpg"
+
+private val mockMovies =
+    listOf(
+        "movie1",
+        "movie2",
+        "movie3",
+        "movie4",
+        "movie5",
+        "movie6",
+    )
+
+private val mockSeries =
+    listOf(
+        "series1",
+        "series2",
+        "series3",
+        "series4",
+        "series5",
+        "series6",
+    )
+
+private val mockAnime =
+    listOf(
+        "anime1",
+        "anime2",
+        "anime3",
+        "anime4",
+        "anime5",
+        "anime6",
+    )
+
+private val mockSoaps =
+    listOf(
+        "soaps1",
+        "soaps2",
+        "soaps3",
+        "soaps4",
+        "soaps5",
+        "soaps6",
+    )
+
+private val mockSpecials =
+    listOf(
+        "specials1",
+        "specials2",
+        "specials3",
+        "specials4",
+        "specials5",
+        "specials6",
+    )
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
+    var currentCategory by rememberSaveable {
+        mutableStateOf(ContentType.MOVIE)
+    }
+
+    val lazyListState = rememberLazyListState()
+
+    val suggestions = remember { mutableStateListOf("Spider-man 2", "Batman") }
+
+    var expanded by rememberSaveable {
+        mutableStateOf(false)
+    }
+
+    val currentItems =
+        when (currentCategory) {
+            ContentType.MOVIE -> mockMovies
+            ContentType.SERIES -> mockSeries
+            ContentType.ANIME -> mockAnime
+            ContentType.SOAPS -> mockSoaps
+            ContentType.SPECIALS -> mockSpecials
+        }
+
+    val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to purple_100,
+                0.6f to black_100,
+            ),
+        )
+
+    LazyColumn(
+        state = lazyListState,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top,
+        modifier = modifier
+            .fillMaxSize()
+            .background(
+                brush = brush,
+            ),
+    ) {
+        item {
+            Spacer(
+                modifier = Modifier.height(64.dp),
+            )
+        }
+        item {
+            HomeScreenHeadline(
+                modifier = Modifier.fillMaxWidth(),
+            )
+        }
+        item {
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+        }
+        item {
+            HomeScreenSearchBar(
+                expanded = expanded,
+                onExpandedChange = {
+                    expanded = it
+                },
+                onSearch = { suggestion ->
+                    if (suggestion.isNotEmpty()) {
+                        suggestions.add(suggestion)
+                    }
+                },
+                suggestions = suggestions,
+            )
+        }
+        item {
+            Spacer(
+                modifier = Modifier.height(36.dp),
```

Comment:
Add this to dimension resource here and in the whole file for every value

Reply:
done